components/Encode/components/EncodeExecute.tsx

```diff
@@ -48,73 +41,81 @@ const EncodeExecute: React.FC<Props> = ({ web3 }) => {
 
   return (
     <>
-      <Grid container justifyContent="center" spacing={2}>
-        <Grid item md={12}>
-          <InputLabel id="demo-simple-select-label">Operation</InputLabel>
-          <Select
-            labelId="demo-simple-select-label"
-            id="demo-simple-select"
-            value={operation}
-            onChange={(event) => setOperation(event.target.value)}
-          >
-            <MenuItem value={'0'}>CALL</MenuItem>
-            <MenuItem value={'1'}>CREATE</MenuItem>
-            <MenuItem value={'2'}>CREATE2</MenuItem>
-            <MenuItem value={'3'}>STATICCALL</MenuItem>
-            <MenuItem value={'4'}>DELEGATECALL</MenuItem>
-          </Select>
-        </Grid>
-        <Grid item md={12}>
-          <TextField
-            label="Recipient"
-            value={recipient}
-            fullWidth
-            onChange={(event) => {
-              const input = event.target.value;
-              setRecipient(input);
-            }}
-          />
-        </Grid>
-        <Grid item md={12}>
-          <TextField
-            label="Amount"
-            value={amount}
-            fullWidth
-            onChange={(event) => {
-              const input = event.target.value;
-              setAmount(input);
-            }}
-          />
-        </Grid>
-        <Grid item md={12}>
-          <TextField
-            label="Data"
-            value={data}
-            fullWidth
-            onChange={(event) => {
-              const input = event.target.value;
-              setData(input);
-            }}
-          />
-        </Grid>
-      </Grid>
+      <div className={styles.gridContainer}>
```

Comment:
I think this can be done with bulma grid system css

Reply:
I tried and it broke the styling due to their [columns](https://bulma.io/documentation/columns/basics/) system.

However, removing the grid and flex styles did not change the format, so I just added the Bluma margin elements.